app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -2186,7 +2186,7 @@ private void updateCurrentBgInfo(String source) {
         }
         if (isWifiWixel || isWifiBluetoothWixel || collector.equals(DexCollectionType.Mock)) {
             updateCurrentBgInfoForWifiWixel(notificationText);
-        } else if (is_follower || collector.equals(DexCollectionType.NSEmulator)) {
+        } else if (is_follower || collector.equals(DexCollectionType.NSEmulator) ||DexCollectionType.isLibreOOPAlgorithm()) {
```

Comment:
Can you instead pass `collector` as a parameter to `isLibreOOPAlgorithm` to avoid repeated look ups of the current collection type.

Reply:
Passed it in both locations.